setup.py

```diff
@@ -31,6 +33,99 @@
         version = match.group(1)
 
 
+def _get_help_output(topic, lines=False):
+    TRANSLATE = {
+        'tar-content': 'tar_content',
+    }
+    if topic in TRANSLATE:
+        topic = TRANSLATE[topic]
+    return _run_snapcraft(['help', topic], lines)
+
+
+def _run_snapcraft(cmd, lines=False):
+    cmd = ['./bin/snapcraft']+cmd
```

Comment:
pep8 doesn't check this file, which is probably a bug. You should use spaces between the operators:
cmd = ['./bin/snapcraft'] + cmd